docs/webpack-dev-server.config.js

```diff
@@ -23,7 +23,7 @@ var config = {
     },
     //Modules will be searched for in these directories
     modulesDirectories: [
-      "web_modules",
```

Comment:
strange... all node_modules of the project are supposed to be included in a 'whatever the current directory is' manner just bellow..